app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/reliefdetails/summary.scala.html

```diff
@@ -53,6 +53,14 @@
           else ""
   }
 }
+
```

Comment:
do the questions in the cya table render properly for period of admin and agent of period of admin? The user key hasn't been updated in this summary template but they've been updated in the PRR and other reliefs template